pandas/core/internals/managers.py

```diff
@@ -406,13 +406,13 @@ def apply(self: T, f, filter=None, **kwargs) -> T:
 
         if f == "where":
             align_copy = True
-            if kwargs.get("align", True):
+            if kwargs.pop("align", True):
```

Comment:
Is this necessary? I think better to not mutate the argument, unless we have full control over where those arguments are coming from

Reply:
We futz with kwargs down on L442 anyway.  We could copy the dict and mess with that, but that doesnt seem better to me

Reply:
in an upcoming PR im going to move the defining of align_keys to the calling function, so "align" can be popped there

Reply:
xref #32846